README.md

```diff
@@ -30,47 +30,47 @@ Supported systems are:
+2. You can download a build from [CI](https://ci.appveyor.com/project/Try/opengothic/history?branch=master) and extract into a folder of your choice. Alternatively, OpenGothic can be built manually. For Arch the [AUR](https://aur.archlinux.org/packages/opengothic) provides a third-party package.
 
-3. Run `Gothic2Notr.sh -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"` (example path, use path to your Gothic installation instead)
+3. Run `Gothic2Notr.sh -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"` (example path, use the path to your Gothic 2 installation instead)
 
    You can edit `Gothic2Notr.sh` and change the line `exec "$DIR/Gothic2Notr" "$@"` to
 
    `exec "$DIR/Gothic2Notr" "$@" -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"`
 
-   to not have to enter path manually every time. Then run `Gothic2Notr.sh` without arguments to start.
+   to not have to enter the path manually every time. Then run `Gothic2Notr.sh` without arguments to start.
 
 ### MacOS
-1. If not already done install Gothic. Instructions how to obtain the game files can be found [here](https://macsourceports.com/faq#getgamedata). OpenGothic comes with auto-path detection if your Gothic files are in `"~/Library/Application Support/OpenGothic"`.
+1. If not already done, install Gothic 2. Instructions on how to obtain the game files can be found [here](https://macsourceports.com/faq#getgamedata). OpenGothic comes with automatic path detection if your Gothic files are in `"~/Library/Application Support/OpenGothic"`.
 2. Download a build from [Mac Source Ports](https://macsourceports.com/game/gothic2) and follow the installation instructions given there.
-   Alternatively recent test builds are available from [CI](https://ci.appveyor.com/project/Try/opengothic/history?branch=master) that can be extracted into a folder of your choice. You can compile a fresh build as well.
+   Alternatively, recent test builds are available from [CI](https://ci.appveyor.com/project/Try/opengothic/history?branch=master) and can be extracted into a folder of your choice. You can compile a fresh build as well.
 3. Run `Gothic2Notr.sh`
 
-   If OpenGothic fails to find your Gothic files you have to explicitly specify its location via `-g` parameter.
-   Change the line `exec "$DIR/Gothic2Notr" "$@"` to reflect your Gothic path e.g.
+   If OpenGothic fails to find your Gothic 2 files, you have to explicitly specify its location via `-g` parameter.
+   Change the line `exec "$DIR/Gothic2Notr" "$@"` to reflect your Gothic 2 path e.g.
 
    `exec "$DIR/Gothic2Notr" "$@" -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"`
 
 ---
 ### Modifications
-Mods can be installed as usual. Provide the `modfile.ini` to OpenGothic via `-game:` parameter to play. Example:
+Mods can be installed as usual. Provide the `modfile.ini` to OpenGothic via the `-game:` parameter to play. Example:
```

Comment:
> the `-game:`

similar to `the path`

Reply:
Here I'd argue again, the parameter is very specific: It is the `-game:` parameter, not any parameter.